src/components/CNav.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <nav class="bg-gray-800 ">
+    <div class="max-w-7xl mx-auto px-2 sm:px-6 lg:px-8">
```

Comment:
responsive is nice but it's not working. Either fix it or remove it (most people will be using a computer to access the tool anyway).